addons/zeus/XEH_postInit.sqf

```diff
@@ -39,4 +41,75 @@ if (isServer) then {
             _x removeCuratorEditableObjects [_objects, true];
         } forEach allCurators;
     }] call CBA_fnc_addEventHandler;
+
+    [QGVAR(createZeus), {
+        params ["_ownerPlayer"];
+        private _owner = ["#adminLogged", getPlayerUID _ownerPlayer] select isMultiplayer;
+        private _group = createGroup sideLogic;
+        private _zeus = _group createUnit ["ModuleCurator_F", [0,0,0], [], 0, "NONE"];
+        missionNamespace setVariable [format [QGVAR(zeus_%1), _owner], _zeus];
+        _zeus setVariable ["owner", _owner, true];
+        _zeus setVariable ["Addons", 3, true];
+        _zeus setCuratorCoef ["Place", 0];
+        _zeus setCuratorCoef ["Delete", 0];
+        _group deleteGroupWhenEmpty true;
+        if (!isMultiplayer && {!isNull findDisplay IDD_DISPLAY3DEN}) then {
+            // if loaded from editor (but not after restart), addons are not activated so we do it manually
+            private _addons = ('true' configClasses (configFile >> "CfgPatches")) apply {configName _x};
+            activateAddons _addons;
+            removeAllCuratorAddons _zeus;
+            _zeus addCuratorAddons _addons;
```

Comment:
I think adding a huge amount of addons causes a major lag on all clients. I never actually tried if the lag spike is gone if I remove the addons but it is my biggest suspect.
Should test that out and if it's really this the addons should be spread over a longer time. Many small lags over 10 seconds are better than half a second game freeze.

Reply:
If you talk about `addCuratorAddons` in multiplayer then ^^ `if (!isMultiplayer`.
If about `_zeus setVariable ["Addons", 3, true]` then nothing we can do and it's mentioned in comments.

Reply:
First one yeah. I always do `addCuratorAddons` in my curator script. Didn't know it wasn't needed.